waddrmgr/address.go

```diff
@@ -785,3 +806,34 @@ func newWitnessScriptAddress(m *ScopedKeyManager, account uint32, scriptHash,
 		isSecretScript: isSecretScript,
 	}, nil
 }
+
+// taprootScriptAddress represents a pay-to-taproot address that commits to a
+// script.
+type taprootScriptAddress struct {
```

Comment:
Can this be removed?

Reply:
This PR isn't ready for review, I'm actively working on stuff.